python/ray/data/preprocessors/encoder.py

```diff
@@ -306,8 +302,7 @@ def column_label_encoder(s: pd.Series):
         return df
 
     def __repr__(self):
-        stats = getattr(self, "stats_", None)
-        return f"LabelEncoder(label_column={self.label_column}, stats={stats})"
+        return f"LabelEncoder(label_column='{self.label_column}')"
```

Comment:
why add colons here?

Reply:
oh it's because its a string

Reply:
what about doing `repr(self.label_column)`?

Reply:
Which colon are you referring to?

Reply:
sorry not colon; apostrophe

Reply:
Oh. Either works, but IMO `'{self.label_column}'` is more readable than `{repr(self.label_column)}`

Reply:
let's do repr please; the colon thing is hard to understand when to apply when extending.